templates/zerver/help/desktop-app-install-guide.md

```diff
@@ -24,12 +23,12 @@ The app will update automatically to future versions.
 
 #### Homebrew
 
-1. Run `brew cask install zulip` in Terminal.
+1. Run the command `brew install --cask zulip` from a terminal.
 
-1. Run Zulip from `Applications`. <!-- TODO fact check -->
+1. Run Zulip from `Applications`.
 
-The app will update automatically to future versions. `brew upgrade` will
-also work, if you prefer.
+The app will update automatically to future versions. You can also run the
+command `brew upgrade zulip` instead.
```

Comment:
"instead" might sound like it has some interaction with the "update automatically" from the previous sentence?

Spitballing, maybe:
```suggestion
The app will update automatically to future versions. Alternately, you can also run the
command `brew upgrade zulip` to immediately upgrade.
```

Reply:
True. Yeah, good idea! except that I think "Alternatively" would be a better word choice because "Alternately" can also mean taking turns. And we can remove "also" so that it's not redundant.